### PR TITLE
Use iptables rpm from a special build

### PR DESCRIPTION
**Pull request by abhat**
Use Iptables build provided by Phil to validate if the segfaults in iptables-save/iptables-restore go away.

### CODE REVIEW

File: images/os/Dockerfile [hunk 1]
```diff
@@ -17,7 +17,11 @@ RUN set -x && yum install -y ostree rpm-ostree yum-utils selinux-policy-targeted
     ostree --repo=/srv/repo checkout -U $commit --subpath /usr/etc/selinux /tmp/tmprootfs/etc/selinux && \
     ostree --repo=/srv/repo commit --parent=$commit --tree=ref=$commit --tree=dir=. \
         --selinux-policy /tmp/tmprootfs \
-        -s "origin-ci-dev overlay RPMs" --branch=origin-ci-dev
+        -s "origin-ci-dev overlay RPMs" --branch=origin-ci-dev && \
+    mkdir /tmp/iptables-rpms && cd /tmp/iptables-rpms && \
+    curl -O http://download.eng.bos.redhat.com/brewroot/work/tasks/8958/27208958/iptables-services-1.8.4-10.el8.x86_64.rpm && \
```
Comment (smarterclayton) [inline images/os/Dockerfile]: You need to ensure the rpm is installed into the ostree image, by downloading the rpm earlier so that ostree comity works.

**Comment by dcbw** [comment]
Also you can't have the RPM on RH-internal systems, you gotta copy it somewhere public:
```
error: could not run steps: step machine-os-content failed: could not wait for build: the build machine-os-content failed after 6m9s with reason DockerBuildFailed: Docker build strategy has failed.

d.eng.bos.redhat.com/brewroot/work/tasks/8958/27208958/iptables-services-1.8.4-10.el8.x86_64.rpm
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --...rl: (6) Could not resolve host: download.eng.bos.redhat.com
```

**Comment by abhat** [comment]
> Also you can't have the RPM on RH-internal systems, you gotta copy it somewhere public:
> 
> ```
> error: could not run steps: step machine-os-content failed: could not wait for build: the build machine-os-content failed after 6m9s with reason DockerBuildFailed: Docker build strategy has failed.
> 
> d.eng.bos.redhat.com/brewroot/work/tasks/8958/27208958/iptables-services-1.8.4-10.el8.x86_64.rpm
>   % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
>                                  Dload  Upload   Total   Spent    Left  Speed
> 
>   0     0    0     0    0     0      0      0 --:--:-- --...rl: (6) Could not resolve host: download.eng.bos.redhat.com
> ```

what is a good public location? Is there some well known place I can point this to?

**Comment by knobunc** [comment]
/hold
Since this is a test...

File: images/os/Dockerfile [hunk 2]
```diff
@@ -1,5 +1,5 @@
 FROM fedora:29 AS build
-
+COPY iptables-services-1.8.4-10.el8.x86_64.rpm /tmp/iptables-rpms
```
Comment (smarterclayton) [inline images/os/Dockerfile]: You have to have a trailing slash

Comment (abhat) [reply images/os/Dockerfile]: Updated the PR.

**Comment by dcbw** [comment]
/retest

gcp job couldn't get a lease

**Pull request closed without merging**